### DISCUSSION TITLE
What parameters to set  for load balancing

### DISCUSSION THREAD

**Discussion by FransOv** [syssi/homeassistant-goecharger-mqtt#87] [Q&A]
What parameter do I need to set in order to limit the maximum current when the total load on the mains connection becomes to high? I see a slider for requested  current (amp), but maximum current limit (ama) is a sensor. From the description in the API I gather that amp gets the value selected by the button on the device whereas ama looks like a better candidate to implement load balancing.

**Comment by kalaws** [comment]
I use `amp`
